third_party/airflow/pyproject.toml

```diff
@@ -9,18 +9,18 @@ requires-python = ">=3.7,<3.9"
 # extremely difficult.
 dependencies = [
 	"armada-client",
-	"apache-airflow~=2.3.1",
+	"apache-airflow>=2.3.1",
 	"grpcio~=1.46.3",
 	"grpcio-tools~=1.46.3",
-	"types-protobuf~=3.19.22"
+	"types-protobuf>=3.19.22"
 ]
 authors = [{name = "Armada-GROSS", email = "armada@armadaproject.io"}]
 license = { text = "Apache Software License" }
 readme = "README.md"
 
 [project.optional-dependencies]
-format = ["black~=22.3.0", "flake8~=4.0.1", "mypy~=0.971"]
-test = ["pytest~=7.1.2", "mypy~=0.971"]
+format = ["black>=22.3.0", "flake8>=4.0.1", "mypy>=0.971"]
```

Comment:
This is OK for now; but could cause (in the future) CI failures related to new flake8/black requirements that we don't meet yet showing up in pypi.